README.md

```diff
@@ -146,6 +146,29 @@ especially in the case in which the exporter is external to the code it's monito
 
 For more detailed documentation on [Availability exporters](https://gitlab.cee.redhat.com/konflux/docs/documentation/-/blob/main/o11y/monitoring/availability_exporters.md?ref_type=heads)
 
+## Recording Rules
+
+Recording rules allow us to precompute frequently needed or computationally expensive expressions
+and save their result as a new set of time series. Recording rules are the go-to approach for
+speeding up the performance of queries that take too long to return. When other teams want to go
+with their own metrics format for exporters they need to adapt to desired metric form by
+translating it using a recording rule. 
+
+These recording rules should be put in the [rhobs/recording folder](https://github.com/redhat-appstudio/o11y/tree/main/rhobs). 
```

Comment:
The link is not pointing to the location mentioned in the link's text.
Also, you can use relative path for links inside the same repo.
See this example: https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file

Reply:
Thanks @yftacherzog! Updated the relative link here